samples/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,119 @@
+.. _suit_flash_companion:
+
+SUIT flash companion
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The flash companion application allows the secure domain firmware to access the external memory during the :ref:`Software Updates for Internet of Things (SUIT) <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+Application overview
+********************
+
+The flash companion implements a device driver for the external flash device and provides an IPC service to the secure domain.
+The secure domain firmware can request from the flash companion to read, erase, or write data to the external memory.
+
+The application is meant to be booted by the secure domain while performing the firmware update process using the :ref:`SUIT <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+The flash companion firmware is not a stand-alone application.
+The application must be built as a child image of a different sample as it inherits the device tree configuration from the parent build.
+
+Requirements
+************
+
+The application supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk/nrf54h20/cpuapp
+
+Configuration
+*************
+
+|config|
+
+Setup
+=====
+
+You can build the application using sysbuild by enabling the :kconfig:option:`SB_CONFIG_SUIT_BUILD_FLASH_COMPANION` Kconfig option.
+The memory partition from which the firmware will run can be configured by providing a device tree overlay through sysbuild. You should create a dedicated partition in non-volatile memory and override the ``zephyr,code-partition``. The memory partition must not be used by anything else.
```

Comment:
```suggestion
The memory partition from which the firmware will run can be configured by providing a device tree overlay through Sysbuild.
You should create a dedicated partition in non-volatile memory and override the ``zephyr,code-partition``.
The memory partition must not be used by anything else.
```

Reply:
Done and replaced other occurrences of `sysbuild` to `Sysbuild`.